browser/src/control/Control.PartsPreview.js

```diff
@@ -171,13 +172,14 @@ L.Control.PartsPreview = L.Control.extend({
 	_createPreview: function (i, hashCode, bottomBound) {
 		var frameClass = 'preview-frame ' + this.options.frameClass;
 		var frame = L.DomUtil.create('div', frameClass, this._partsPreviewCont);
-		frame.id = 'preview-frame-part-' + i;
+		frame.id = 'preview-frame-part-' + this._idNum;
```

Comment:
why do we need this additional `_idNum` variable?  wasn't  `i` already doing the same?
what when we switch to master page and go back? or we add/delete slide?
I think this function will be called again but index will not be reset